<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusyDialog</class>
 <widget class="QDialog" name="BusyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Please Wait...</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { background-color : rgb(230, 240, 249);
					border-width: 1px; border-style: solid;
					border-color:   #D8D8D8;};</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>22</y>
     <width>241</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <item row="0" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QDilaog {background-color : transparent};</string>
      </property>
      <property name="text">
       <string>Please Wait...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="BusyIndicator" name="busyIndicator0">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="BusyIndicator" name="busyIndicator1">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="BusyIndicator" name="busyIndicator2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="BusyIndicator" name="busyIndicator3">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="BusyIndicator" name="busyIndicator4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="BusyIndicator" name="busyIndicator5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="BusyIndicator" name="busyIndicator6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BusyIndicator</class>
   <extends>QLabel</extends>
   <header location="global">Waiting/BusyIndicator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
